config/initializers/sentry.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require_relative '../../lib/util/git'
+
+# ENV['SENTRY_DSN'] is reserved by Sentry and its use seems to prevent this initializer from being recognized.
+# Hence, we use CAS_SENTRY_DSN. Any other alternate key should also be fine.
+if ENV['CAS_SENTRY_DSN'].present?
+  Sentry.init do |config|
+    config.dsn = ENV['CAS_SENTRY_DSN']
+    config.breadcrumbs_logger = [:active_support_logger, :http_logger]
+
+    config.enabled_environments = ['production', 'staging', 'development'] # Remove development when dev is done.
```

Comment:
Do we want to remove dev now?

Reply:
Not if we want to capture errors from local dev environments. I put that comment in before I realized that was a use case. I'll remove it (the comment not `development`)